health/micro-ui/web/micro-ui-internals/packages/modules/microplan/src/pages/employee/PlanWithCensusInbox.js

```
@@ -0,0 +1,400 @@
+                setActiveLink(e);
+              }}
+              setActiveLink={setActiveLink}
+              showNav
+              style={{}}
+            />
+          )}
+          <Card className="microPlanBulkTable" type={"primary"}>
+            {villagesSlected !== 0 && (
+              <div className="selection-state-wrapper">
+                <div className="svg-state-wrapper">
+                  <SVG.DoneAll width={"1.5rem"} height={"1.5rem"} fill={"#C84C0E"}></SVG.DoneAll>
```

Biome

[error] JSX elements without children should be marked as self-closing. In JSX, it is valid for any element to be self-closing.

Unsafe fix: Use a SelfClosingElement instead


(lint/style/useSelfClosingElements)